packages/css/avatar.css

```diff
@@ -0,0 +1,93 @@
+.ds-avatar {
+  --ds-avatar-background: var(--ds-color-accent-base-default);
+  --ds-avatar-color: var(--ds-color-accent-contrast-default);
+  --ds-avatar-size: var(--ds-sizing-12);
+  --ds-avatar-padding: var(--ds-spacing-2);
+  --ds-avatar-border-radius: 50%;
+
+  background-color: var(--ds-avatar-background);
```

Comment:
Could this be just `background` so it correlates with the css custom property `--ds-avatar-background`? ☺️

Reply:
I don't have any opinion on either, so sure :)